elements/nuxeo-results/nuxeo-results.html

```diff
@@ -377,15 +379,18 @@
       },
 
       fetch: function() {
-        return new Promise(function(resolve, error) {
-          this.debounce('fetch', function() {
+        return new Promise((resolve, error) => {
+          this._fetchDebouncer = Polymer.Debouncer.debounce(this._fetchDebouncer, Polymer.Async.timeOut.after(100), () => {
```

Comment:
On 10.10, to keep consistency, we should avoid ES6 syntax.

```suggestion
          this._fetchDebouncer = Polymer.Debouncer.debounce(this._fetchDebouncer, Polymer.Async.timeOut.after(100), function() => {
```

Reply:
Thank you 👍